ReactAndroid/build.gradle

```diff
@@ -337,8 +337,8 @@ dependencies {
     testImplementation("org.robolectric:robolectric:${ROBOLECTRIC_VERSION}")
 
     androidTestImplementation(fileTree(dir: "src/main/third-party/java/buck-android-support/", include: ["*.jar"]))
-    androidTestImplementation("com.android.support.test:runner:${ANDROID_SUPPORT_TEST_VERSION}")
-    androidTestImplementation("com.android.support.test:rules:${ANDROID_SUPPORT_TEST_VERSION}")
+    androidTestImplementation("androidx.test:runner:${ANDROID_SUPPORT_TEST_VERSION}")
```

Comment:
Change this to `1.1.0` ?
https://developer.android.com/jetpack/androidx/releases/test